eu.nokun.MirrorHall.json

```diff
@@ -0,0 +1,76 @@
+{
+    "app-id": "eu.nokun.MirrorHall",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "47",
+    "add-extensions": {
+        "org.freedesktop.Platform.ffmpeg-full": {
+            "version": "24.08",
+            "directory": "lib/ffmpeg",
+            "add-ld-path": ".",
+            "no-autodownload": false,
+            "autodelete": false
+        },
+        "org.freedesktop.Platform.GStreamer.gstreamer-vaapi": {
+            "directory": "lib/gstreamer-1.0",
+            "version": "24.08",
+            "autodelete": false,
+            "no-autodownload": true,
+            "add-ld-path": "lib",
+            "download-if": "have-intel-gpu",
+            "autoprune-unless": "have-intel-gpu"
+        }
+    },
+    "sdk": "org.gnome.Sdk",
+    "command": "mirrorhall",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--device=dri",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=xdg-run/pipewire-0",
+        "--talk-name=org.gnome.Mutter.ScreenCast",
+        "--env=GST_DEBUG=2",
+        "--env=GST_PLUGIN_PATH=/app/lib/gstreamer-1.0:/usr/lib/gstreamer-1.0"
```

Comment:
`/usr/lib/gstreamer-1.0` doesn't exist. What you want is to put `/app` and the extension path, but that's already done via the runtime.

```
flatpak run --command=env org.gnome.Platform//46|grep GST_PLUGIN_SYSTEM_PATH
GST_PLUGIN_SYSTEM_PATH=/app/lib/gstreamer-1.0:/usr/lib/extensions/gstreamer-1.0:/usr/lib/x86_64-linux-gnu/gstreamer-1.0
```